frontend/src/scenes/feature-flags/FeatureFlagSnippets.tsx

```diff
@@ -337,10 +337,24 @@ export function AndroidSnippet({ flagKey, multivariant, payload }: FeatureFlagSn
     )
 }
 
-export function FlutterSnippet({ flagKey }: FeatureFlagSnippet): JSX.Element {
+export function FlutterSnippet({ flagKey, multivariant, payload }: FeatureFlagSnippet): JSX.Element {
+    const clientSuffix = 'await Posthog().'
+
+    if (payload) {
+        return (
+            <CodeSnippet language={Language.Dart} wrap>
+                {`${clientSuffix}getFeatureFlagPayload('${flagKey}')`}
+            </CodeSnippet>
+        )
+    }
+
+    const flagFunction = multivariant ? 'getFeatureFlag' : 'isFeatureEnabled'
+
+    const variantSuffix = multivariant ? ` == 'example-variant'` : ''
+
     return (
         <CodeSnippet language={Language.Dart} wrap>
-            {`if (await Posthog().isFeatureEnabled('${flagKey}') ?? false) {
+            {`if (${clientSuffix}${flagFunction}('${flagKey}') ${variantSuffix}) {
```

Comment:
We shouldn't need the extra space because we've accounted for it in `variantSuffix`
```suggestion
            {`if (${clientSuffix}${flagFunction}('${flagKey}')${variantSuffix}) {
```

Reply:
good catch, also fixed for android and ios